test/suites/scheduling/suite_test.go

```diff
@@ -800,13 +801,58 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 				})
 				if ok {
 					reservedCount += 1
-					Expect(req.Values).To(ConsistOf(xlargeCapacityReservationID))
+					Expect(req.Values).To(ConsistOf(largeCapacityReservationID))
 				}
 			}
 			Expect(reservedCount).To(Equal(1))
 			env.EventuallyExpectNodeCount("==", 2)
 		})
 		It("should demote reserved instances when the reservation is canceled", func() {
+			var canceled bool
```

Comment:
This is also covered by drift -- does it need to be in both?

Reply:
Yeah, we can probably drop this from the drift test now that it's handled in scheduling.